packages/core/src/+internal/utils/any.util.ts

```diff
@@ -20,3 +32,15 @@ export const wait = (seconds = 1) =>
   new Promise(res => {
     setTimeout(() => res(), seconds * 1000);
   });
+
+export const bufferFrom = (data: any): Buffer =>
+  Buffer.from(data);
+
+export const transformUrlEncoded = (data: any): string =>
+  !isString(data) ? qs.stringify(data) : data;
+
+export const stringifyJson = (data: any): string =>
+  !isString(data) ? JSON.stringify(data) : data;
+
+export const parseJson = (data: any) =>
+  isJsonParseable(data) ? JSON.parse(data) : data;
```

Comment:
Its nice for readability, but is it really necessary to `JSON.parse` twice?

Reply:
Yeah... good spot. 👍